src/components/Profile/Profile.tsx

```diff
@@ -42,24 +53,30 @@ const BlurbContainer = styled.div`
   }
 `
 
-const FooterContainer = styled.div`
+const FooterContainer = styled(Box)`
   grid-column: 1 / 13;
   grid-row: 5 / 7;
 `
 
-const Profile = ({ avatar }: any) => (
-  <IntroWrapper className="items-center justify-center pv4">
-    <AvatarContainer className="center w-100-ns w-50">
-      <Avatar image={avatar} />
-      <BlackMirror />
-    </AvatarContainer>
-    <BlurbContainer className="tc tr-ns ph2">
-      <ProfileBlurb />
-    </BlurbContainer>
-    <FooterContainer className="bt b--light-silver">
-      <ProfileFooter />
-    </FooterContainer>
-  </IntroWrapper>
-)
+const Profile = ({ avatar }: any) => {
+  return (
+    <IntroWrapper justifyContent="center" alignItems="center" mt={4}>
+      <AvatarContainer
+        ml={['25%', '0']}
+        width={['50%', '100%']}
+        className="center"
+      >
+        <Avatar image={avatar} />
+        <BlackMirror />
+      </AvatarContainer>
+      <BlurbContainer className="tc tr-ns ph2" pb="4">
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 12 spaces but found 6. (@typescript-eslint/indent)](https://app.codacy.com/app/jdstuart/jesses.io/pullRequest?prid=4576429)